iris_grib/message.py

```diff
@@ -33,6 +33,8 @@
 from iris._lazy_data import as_lazy_data
 from iris.exceptions import TranslationError
 
+SUPPORTED_GRID_DEFINITIONS = (0, 1, 5, 10, 12, 20, 30, 40, 90)
```

Comment:
What benefit did you get for factoring this out to the top of the file? I don't see it's a problem, I'm just interested...

You could probably do with making it a private global variable though (add a leading underscore `_`), as per the similar variables in `_load_convert`.